addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -299,7 +299,7 @@ export class PaymentScreen extends Component {
         if (
             syncOrderResult &&
             syncOrderResult.length > 0 &&
-            this.currentOrder.wait_for_push_order()
+            syncOrderResult.some((order) => order.wait_for_push_order())
```

Comment:
This method is just checkout some condition, the name is strange. maybe rename it in master :)